src/lib/pages/backoffice/Providers/ProviderEditor/schema.js

```diff
@@ -0,0 +1,30 @@
+export const schema = {
+  properties: {
+    address: {
+      type: 'string',
+      title: 'Address',
+    },
+    email: {
+      type: 'string',
+      title: 'Email',
+    },
+    name: {
+      type: 'string',
+      title: 'Name',
+    },
+    notes: {
+      type: 'string',
+      title: 'Notes',
+    },
+    phone: {
+      type: 'string',
+      title: 'Phone',
+    },
+    type: {
+      type: 'string',
+      title: 'Type',
+    },
+  },
+  required: ['name', 'type'],
```

Comment:
the `type` is required I think in the backend.

Reply:
yes that's why I have it as required here too. Am I missing something?

Reply:
No it is me that I didn't see it sorry... maybe it was too late :'(